src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElse(null);
```

Comment:
It's bad practice to use `orElse(null)`. It is better to get an exception in the place where it occurred than get an NPE in completely different program service and take a long time to figure out where it came from.
```suggestion
        return driverDao.get(id).get();
```

Reply:
i will handle NoSuchElementException in service. Ok?